assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -567,10 +574,24 @@ che.server.secure_exposer.jwtproxy.auth.loader.path=/_app/loader.html
 che.server.secure_exposer.jwtproxy.image=eclipse/che-jwtproxy:latest
 che.server.secure_exposer.jwtproxy.memory_limit=128mb
 
-# Maximum size of the json processing pool
-# in case if pool size would be exceeded message execution will be rejected
-che.core.jsonrpc.processor_max_pool_size=100
 
+## Configuration of major "/websocket" endpoint
+# Maximum size of the JSON RPC processing pool
+# in case if pool size would be exceeded message execution will be rejected
+che.core.jsonrpc.processor_max_pool_size=50
+# Initial json processing pool. Minimum number of threads that used to process major JSON RPC messages.
+che.core.jsonrpc.processor_core_pool_size=3
+# Configuration of queue used to process Json RPC messages. If >0 capacity of LinkedBlockingQueue, <=0 SynchronousQueue
```

Comment:
I think 
`If >0 capacity of LinkedBlockingQueue, <=0 SynchronousQueue` part may be improved.
something like 
`If the configured value >0 then it's used as a capacity of LinkedBlockingQueue, otherwise (the value <=0) it is used as a capacity of SynchronousQueue`